examples/api/lib/material/context_menu/anywhere.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file should be named to match the snake_case version of the symbol that it is attached to in the `context_menu.dart` file, along with an index of which example on that symbol this is. This makes it easy to add more than one example to a symbol and keep track of them.

```
examples/api/lib/material/context_menu/context_menu_controller.0.dart
```
Same for all the examples (which are really great to see!).

Reply:
Thanks for the explanation, I thought I might have been doing that wrong.

Are the tests supposed to be like context_menu.0_test.dart or context_menu.0.test.dart?  I see both patterns multiple times in the examples test directory.

Reply:
`context_menu.0_test.dart`, since otherwise they're not considered tests by some tools (most notably, I think `@visibleForTesting` doesn't work then).